File: frontend/RecipeApp.WinForms/Models/ManufacturerDto.cs
namespace RecipeApp.WinForms.Models;

public class ManufacturerDto
{
    public string Name { get; set; } = string.Empty;
    public string Country { get; set; } = string.Empty;
    public string? LegalAddress { get; set; }
    public string? Inn { get; set; }
    public string? Ogrn { get; set; }
    public string? ContactPhone { get; set; }
    public string? ContactEmail { get; set; }
    public string? Website { get; set; }
}

File: src/main/java/com/aksenova/additivesmanager/dto/ManufacturerDto.java
package com.aksenova.additivesmanager.dto;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class ManufacturerDto {

    @NotBlank(message = "name is required")
    private String name;

    @NotBlank(message = "country is required")
    private String country;

    private String legalAddress;
    private String inn;
    private String ogrn;
    private String contactPhone;

    @Email(message = "contactEmail must be a valid email")
    private String contactEmail;

    private String website;
}
